### PR TITLE
Fix ARMv7 process detection

### PR DESCRIPTION
**Pull request by Biswa96**

### CODE REVIEW

File: ProcessHacker/proctree.c [hunk 1]
```diff
@@ -3276,7 +3276,7 @@ BOOLEAN NTAPI PhpProcessTreeNewCallback(
                             PhInitializeStringRef(&getCellText->Text, L"x64");
                             break;
                         }
-                    case IMAGE_FILE_MACHINE_ARM:
+                    case IMAGE_FILE_MACHINE_ARMNT:
```
Comment (jxy-s) [inline ProcessHacker/proctree.c:3279]: probably just add `IMAGE_FILE_MACHINE_ARMNT`:

```cpp
case IMAGE_FILE_MACHINE_ARM:
case IMAGE_FILE_MACHINE_ARMNT:
```

Comment (Biswa96) [reply ProcessHacker/proctree.c:3279]: Would not that be incorrect? According to the [docs][1],

> All code executed on this platform is expected to start and always remain in Thumb mode


[1]: https://docs.microsoft.com/en-us/cpp/build/overview-of-arm-abi-conventions

Comment (jxy-s) [reply ProcessHacker/proctree.c:3279]: Yeah you're right, this is fine. Thank for fixing it.

**Pull request merged**